pkg/cli/debug_recover_loss_of_quorum.go

```diff
@@ -104,7 +104,39 @@ recovery steps to verify the data and ensure database consistency should be
 taken ASAP. Those actions should be done at application level.
 
 'debug recover' set of commands is used as a last resort to perform range
-recovery operation. To perform recovery one should perform this sequence
+recovery operation.
+
+Loss of quorum recovery could be performed in two modes half-online and
+offline. Half-online approach is a preferred one but offline approach is
+preserved for compatibility with any existing tooling that may exist. The
+main difference between approaches is how information in cluster is collected
```

Comment:
I'd argue that the main difference (and advantage) is that online doesn't require a cluster shutdown.